packages/stream_chat_flutter_core/lib/src/message_list_core.dart

```diff
@@ -134,8 +134,7 @@ class MessageListCoreState extends State<MessageListCore> {
 
     bool defaultFilter(Message m) {
       final isMyMessage = m.user?.id == _currentUser?.id;
-      final isDeletedOrShadowed = m.isDeleted == true || m.shadowed == true;
-      if (isDeletedOrShadowed && !isMyMessage) return false;
+      if (m.shadowed && !isMyMessage) return false;
```

Comment:
are we showing deleted messages to all the users? Or only for current user

Reply:
all users